Models.md

```diff
@@ -0,0 +1,48 @@
+# CS Curricular Planning Data Modeling
+
+## Course Models
+
+### Semester
+* `academic_year` : string
+* `semester` : string (ex. fall or spring)
+* `courses_offered`: [Offering]
+
+### Course
+* `number` : integer/string
```

Comment:
Is this supposed to be like a unique identifier? How is it different from `course_id`?